content/_startups/gap.md

```diff
@@ -8,9 +8,10 @@ phases:
     start: 2020-10-13 # date au format ISO (AAAA-MM-DD)
     end: 2021-03-01 # 
 events: # les évènements marquants de la startup - les noms possibles sont définies dans /content/_events/
-  - name: committee
```

Comment:
@be-mercier on ne supprime pas les évenements.

Aussi alumni va dans la section "phases"